process.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaeger
+
+// Process holds process specific metadata that's relevant to this client.
+type Process struct {
+	Service string
+	UUID    string
```

Comment:
Why is this called `UUID`? 
We only seem to be using it as an arbitrary identifier.

Reply:
At one point we actually did use a UUID. Not sure what happened.

Reply:
I don't think we should continue calling this field UUID if it's not going to be holding a UUID.

Reply:
See #248

Reply:
It will be a UUID, I just have to combine this with https://github.com/jaegertracing/jaeger-client-go/pull/248/files